.github/dependabot.yml

```diff
@@ -17,3 +17,18 @@ updates:
     schedule:
       interval: "weekly"
     open-pull-requests-limit: 10
+    groups:
+      # Production dependencies without breaking changes
+      dependencies:
+        dependency-type: "production"
+        update-types:
+        - "minor"
+        - "patch"
+      # Production dependencies with breaking changes
+      dependencies-major:
+        dependency-type: "production"
+        update-types:
+        - "major"
```

Comment:
I don't think we should group major production dep changes, as it may make debugging breaking changes harder if there's multiple coming in from different deps?

Reply:
I assume it is very likely to be two major bump on the production dependencies at the same time.
Most of the time, we do a manual change way before dependabot can pick-up.